### ISSUE TITLE
ppyoloe-s的官网量化训练为什么这么慢呢

### ISSUE THREAD

**Issue by sdreamforchen** [PaddlePaddle/PaddleDetection#7838]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

ppdet.engine INFO: Epoch: [0] [  400/14658] learning_rate: 0.000055 loss: 2.164398 loss_cls: 1.239831 loss_iou: 0.191094 loss_dfl: 0.920798 loss_l1: 0.465173 eta: 215 days, 8:32:25 batch_cost: 4.4311 data_cost: 0.0002 ips: 1.8054 images/s
采用如下命令进行量化训练，为什么训练速度很慢呢？GPU使用率也很低。：
python tools/train.py -c /configs/ppyoloe/ppyoloe_crn_s_300e_coco.yml --slim_config /configs/slim/post_quant/ppyoloe_crn_s_300e_coco_ptq.yml
PS：进行了网络模型训练，速度正常

**Comment by lyuwenyu** [comment]
导出的模型用预训练了嘛

**Comment by sdreamforchen** [comment]
没有导出模型，config里面用的是paddle训练完后的模型



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年3月13日(周一) 下午5:26
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PaddlePaddle/PaddleDetection] ppyoloe-s的官网量化训练为什么这么慢呢 (Issue #7838)




 
导出的模型用预训练了嘛

**Comment by sdreamforchen** [comment]
Paddledetection官网下载的



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年3月13日(周一) 下午5:26
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PaddlePaddle/PaddleDetection] ppyoloe-s的官网量化训练为什么这么慢呢 (Issue #7838)




 
导出的模型用预训练了嘛
